docs/modules/ROOT/pages/type-definitions/schema-configuration/type-configuration.adoc

```diff
@@ -0,0 +1,238 @@
+[[schema-configuration-type-configuration]]
+= Type Configuration
+
+When representing a Neo4j Node, a GraphQL Object type produces multiple operation fields in the `Query`, `Mutation`, and `Subscription` types. 
+For example:
+
+[source, graphql, indent=0]
+----
+type Movie { 
+    title: String 
+    length: Int
+}
+----
+
+From these type definitions, the library generates the following operation fields:
+
+**Query**:
+
+    * `movies`
+    * `moviesAggregate`
+    * `moviesConnection`
+
+**Mutation**:
+
+    * `createMovies`
+    * `deleteMovies`
+    * `updateMovies`
+
+**Subscription**:
```

Comment:
Had a chat with @angrykoala about it, seems that the part of the migration guides where we describe these changes on the current subscription plugin is missing. I don't know if it will be better to include it in this PR or in a separate one tho.